charts/bundler/templates/deployment.yaml

```diff
@@ -68,6 +70,13 @@ spec:
               port: http
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+            - name: static-bundler-volume
+              mountPath: {{ .Values.bundler.static_data }}
```

Comment:
This should match the key in the `values.yaml` (i.e. `.Values.bundler.staticData`)